docs/deployment-architecture.html

```
@@ -0,0 +1,545 @@
+      ESP["ESP8266 / 센서<br/>(LAN)"]:::local
+    end
+
+    subgraph External["🌐 외부 SaaS"]
+      RunPod["RunPod GPU<br/>파인튜닝 전용"]:::ext
+      Groq["Groq Whisper STT"]:::ext
+      OpenAPI["공공 API<br/>기상/농약/KAMIS"]:::ext
+      Discord["Discord 웹훅<br/>알림 수신"]:::ext
+      GitHub["GitHub Actions<br/>CI/CD"]:::ext
+    end
+
+    User -->|"HTTPS"| R53
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)